tests/queries/0_stateless/03594_text_index_size_in_data_skipping_indices.sql

```diff
@@ -0,0 +1,37 @@
+SET allow_experimental_full_text_index = 1;
+
+DROP TABLE IF EXISTS tab;
+CREATE TABLE tab
+(
+    str String,
+    INDEX text_idx str TYPE text(tokenizer = 'ngram', ngram_size = 3) GRANULARITY 1,
+    INDEX set_idx str TYPE set(10) GRANULARITY 1
+)
+ENGINE = MergeTree
+ORDER BY tuple()
+SETTINGS compress_marks = 0;
```

Comment:
Wondering if `compress_marks = 0` is required for the test to succeed?

Reply:
Yes, without this setting the test fails non-deterministically. It seems sometimes marks are compressed and sometimes not (perhaps has something to do with how tiny this table is)